sdk/core/azure-core/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## 1.0.0-beta.7 (Unreleased)
 
+### Breaking Changes
+
+- Moved `Azure::Core::Http::HttpPipeline` to the internal layer `Azure::Core::Internal::Http::HttpPipeline`.
```

Comment:
From the end-customer perspective, this API is now gone. Let's highlight that part.
```suggestion
- Removed `Azure::Core::Http::HttpPipeline` by making it internal, used only within the SDK.
```